dd-sdk-android/src/main/json/rum/view-schema.json

```diff
@@ -199,16 +194,69 @@
                 }
               },
               "readOnly": true
+            },
+            "in_foreground_periods": {
```

Comment:
Are we computing this for now ?

Reply:
No, it's part of the global rum schemas (it's used by the browser)